docker/docker-compose-local.yaml

```diff
@@ -86,12 +88,30 @@ services:
       - frontend-next:/home/owasp/.next
       - frontend-node-modules:/home/owasp/node_modules
 
+  redis:
+    container_name: nest-redis
+    image: redis:7.2-alpine
```

Comment:
Could you pin this version to the patch value, e.g. `redis:7.2.7-alpine3.21` -- use 7.2.? instead of just 7.2